timesketch/ui/css/ts.scss

```diff
@@ -230,11 +230,19 @@ header {
     height:34px;
     border:1px solid #d1d1d1;
     width:150px;
+    outline: none;
 }
 
 .select-view {
     margin-left:5px;
-    outline: none;
+}
+
+.select-graph-view {
```

Comment:
Please be consistent with other css related to graph part, please nest it inside component selector:
```scss
ts-graphs-cypher-query {
    .#{$}__predefined-queries {
        ...
    }
}
```

Reply:
Done.